packages/chargebee/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
you didn't add anything, therefore patch version bump?

Reply:
i did bump peers, but eh not sure, let's try patch version

Reply:
breaking peer dump -> major
nothing breaking -> patch

why minor?